.circleci/config.yml

```diff
@@ -238,6 +238,10 @@ jobs:
           name: Run e2e file upload extensions test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="fileUploadExtensions"
+      - run:
+          name: Run e2e contributor dashboard test
```

Comment:
I have to move this test from Travis to circleCI as this suite performs file-uploads, so it should run in non-prod mode.

Reply:
Can we create a non-prod test suite in travis instead? In general I think we want to move out of Circle as much as possible

Reply:
Actually Travis doesn't handle non-prod mode well, IIRC. There is some issue with accessing the local GCS file storage -- for some reason, our calls to that end up going to Travis's GCS file storage instead. So we might have no choice...

Reply:
Ah okay if that's the case this makes sense! Approving then.